dali/ft/filecopy.hpp

```diff
@@ -132,7 +132,7 @@ static const char * DfuOperatonStr[] =
         "DFUreplicate",
         "DFUtransfer"
     };
-interface IFileSprayer : public IInterface
+interface IFileSprayer : public IInterface, IAbortRequestCallback
```

Comment:
Isn't necessary to extend the IFileSprayer interface and doesn't really make sense for it to be a extension of IAbortRequestCallback, as it itself takes a IAbortRequestCallback.

Instead you can implement a IAbortRequestCallback inside FileSprayer and use it for the partitioner->setAbort calls, e.g.:
```
    class CAbortRequestCallback : implements IAbortRequestCallback
    {
        FileSprayer &sprayer;
    public:
        CAbortRequestCallback(FileSprayer &_sprayer) : sprayer(_sprayer) { }
        virtual bool abortRequested() { return sprayer.isAborting(); }
    } fileSprayerAbortChecker;
```

can make it a member class of FileSprayer